metadata.json

```diff
@@ -36,5 +32,11 @@
         "7"
       ]
     }
+  ],
+  "requirements": [
+    {
+      "name": "puppet",
+      "version_requirement": ">= 4.4.0 < 5.0.0"
```

Comment:
puppet didn't accept http/https URLS in the source parameter of the file type before version [4.4](http://ffrank.github.io/features/2016/02/06/using-http-files/)